network-api/networkapi/mozfest/migrations/0009_alter_mozfestprimarypage_search_image.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.13 on 2022-10-13 17:43
```

Comment:
A more descriptive migration file name would be nice.

Reply:
Updated to [0009_update_searchimage_label_and_help_text.py](https://github.com/mozilla/foundation.mozilla.org/blob/da8b6fb62a2bddab24913644d481b502f5d702e3/network-api/networkapi/mozfest/migrations/0009_update_searchimage_label_and_help_text.py)!